pytg/__init__.py

```diff
@@ -83,7 +87,15 @@ def preexec_function():
                 raise TypeError("custom_cli_args should be a list or a tuple.")
             args.extend(custom_cli_args)
         logger.info("Starting Telegram Executable: \"{cmd}\"".format(cmd=" ".join(args)))
-        self._proc = subprocess.Popen(args, stdin=subprocess.PIPE, stdout=subprocess.PIPE, preexec_fn=preexec_function)
+
+        if self.home_dir:
+            logger.info("Using custom Telegram home directory", self.home_dir)
+            custom_env = os.environ.copy()
+            custom_env["TELEGRAM_HOME"] = self.home_dir
+            self._proc = subprocess.Popen(args, stdin=subprocess.PIPE, stdout=subprocess.PIPE,
+                                          preexec_fn=preexec_function, env=custom_env)
+        else:
+            self._proc = subprocess.Popen(args, stdin=subprocess.PIPE, stdout=subprocess.PIPE, preexec_fn=preexec_function)
         if self._check_stopped():
```

Comment:
Now there are to almost identical subprocess calls.
Can we just have the variable `custom_env` set, or default to `None`? 

Something like
```py
custom_env = None
if self.home_dir:
  logger.info("Using custom Telegram home directory", self.home_dir)
  custom_env = os.environ.copy()
  custom_env["TELEGRAM_HOME"] = self.home_dir
# end if
self._proc = subprocess.Popen(args, stdin=subprocess.PIPE, stdout=subprocess.PIPE,  env=custom_env)
```
?

Reply:
Yep, looks more readable this way.